src/modules/wallets/RPCWallet.js

```diff
@@ -688,6 +704,85 @@ class RPCWallet {
     // Return the TXID of the transaction
     return broadcastTX.result
   }
+
+  /**
+   * Subscribe a callback function to be run when all TXIDs are confirmed.
+   * @param  {Array<String>} response.txids - An Array of TXIDs
+   * @param  {OIPRecord} response.record - The OIPRecord that was published (to be returned to the Callback)
+   * @param {function} options.onConfirmation - A function to run once the transaction recieves a confirmation that it was included in a block.
+   * @param {String} options.onConfirmationRef - A reference string that should be returned in the onConfirmation function (useful for ID's).
+   */
+  onConfirmation ({ txids, record }, options) {
+    // If we do not have an onConfirmation subscription, exit the function
+    if (!options || !options.onConfirmation || typeof options.onConfirmation !== 'function') { return }
```

Comment:
Ha ha ha, love the multiple checks for the onConfirmation function.

Reply:
Gotta love Javascript!